pkgs/tools/networking/nfstrace/default.nix

```diff
@@ -0,0 +1,22 @@
+{ cmake, fetchFromGitHub, gcc7Stdenv, libpcap, stdenv }:
+
+gcc7Stdenv.mkDerivation rec {
+  pname = "nfstrace";
+  version = "0.4.3.2";
+
+  src = fetchFromGitHub {
+    owner = "epam";
+    repo = "nfstrace";
+    rev = "${version}";
+    sha256 = "sha256:1djsyn7i3xp969rnmsdaf5vwjiik9wylxxrc5nm7by00i76c1vsg";
+  };
+
+  buildInputs = [ cmake libpcap ];
```

Comment:
`cmake` should be a `nativeBuildInput`, please add `nativeBuildInputs = [ cmake ];` before this line and remove `cmake` from `buildInputs`.

Reply:
thanks a lot for the quick review @dtzWill 
fixed it in dacbc00686c and rebased it back into a single commit.